docs/cloud-controller-manager.md

```diff
@@ -33,33 +36,116 @@ To use cloud controller manager, the following components need to be configured:
 
     |Flag|Value|Remark|
     |---|---|---|
-    |--cloud-provider|azure||
-    |--cloud-config||Path for [cloud provider config](cloud-provider-config.md)|
-    |--kubeconfig||Path for cluster kubeconfig|
+    |--cloud-provider|azure|cloud-provider should be set azure|
+    |--cloud-config|/etc/kubernetes/azure.json|Path for [cloud provider config](cloud-provider-config.md)|
+    |--kubeconfig|/etc/kubernetes/kubeconfig|Path for cluster kubeconfig|
 
     For other flags such as `--allocate-node-cidrs`, `--configure-cloud-routes`, `--cluster-cidr`, they are moved from kube-controller-manager. If you are migrating from kube-controller-manager, they should be set to same value.
 
     For details of those flags, please refer to this [doc](https://kubernetes.io/docs/reference/command-line-tools-reference/cloud-controller-manager/).
 
 Alternatively, you can use [aks-engine](https://github.com/Azure/aks-engine) to deploy a Kubernetes cluster running with cloud-controller-manager. It supports deploying `Kubernetes azure-cloud-controller-manager` for Kubernetes v1.8+.
 
+## AzureDisk and AzureFile
+
+AzureDisk and AzureFile volume plugins are not supported with external coud provider (See [kubernetes/kubernetes#71018](https://github.com/kubernetes/kubernetes/issues/71018) for explanations).
+
+Hence, [azuredisk-csi-driver](https://github.com/kubernetes-sigs/azuredisk-csi-driver) and [azurefile-csi-driver](https://github.com/kubernetes-sigs/azurefile-csi-driver) should be used for persistent volumes.
+
+### Deploy AzureDisk CSI plugin
+
+Run following commands:
```

Comment:
add a deployment link may be more better

Reply:
Added links for each driver.